packages/poi-preset-babel-minify/index.js

```diff
@@ -4,8 +4,11 @@ module.exports = (pluginOptions, overrides) => {
       // Say goodbye to uglify plugin
       const BabelMinifyPlugin = require('babel-minify-webpack-plugin')
 
-      config.plugin('minimize')
-        .use(BabelMinifyPlugin, [pluginOptions, overrides])
+      // do not use if `minimize` is off
+      if (poi.options.minimize) {
```

Comment:
`minimize` might actually be `undefined` in `production` mode, but we still need to minimize it. We can use `if (config.plugins.has('minimize'))` instead.

Reply:
`config.plugins.has('minimize')` will resolve as `true` even if `minimize` is `false`.

I think an explicit check should do it: `if (poi.options.minimize !== false)`

Reply:
Good point. Changed.